### ISSUE TITLE
DAN error report

### ISSUE THREAD

**Issue by VoiceBeer** [easezyc/deep-transfer-learning#9]
Hi, thx for ur code.

There's an error when I run the DAN code. 

The log can be seen here:
![image](https://user-images.githubusercontent.com/18626699/78978896-4c3d3780-7b4d-11ea-9f70-5f451a024879.png)

The only thing I edited in the code is changing the root_path from `/data/zhuyc/OFFICE31/` to `./dataset/Original_images/`(except for the print log with "tt" and "ttt"). I checked the size of source data and target data but nothing wrong, both are `[32, 3, 224, 224]`. Is there any mistake in the code or did I run the wrong way?

Thank you in advance and have a good day :>

**Comment by easezyc** [comment]
I did not meet this question. I suggest that you should：
print(source.size())
source = self.cls_fc(source)

**Comment by VoiceBeer** [comment]
Sorry about replying u this late, I edited `ResNet.py: Line 177` from `self.cls_fc = nn.Linear(2048, num_classes)` to `self.cls_fc = nn.Linear(1000, num_classes)`  and it works. Thank you very much, but why the previous version of this line is 2048?

**Comment by easezyc** [comment]
I think you have changed my code. The output of the last conv of Resnet50 should be 2048. And the self.cls_fc = nn.Linear(2048, num_class) is right.

**Comment by VoiceBeer** [comment]
Actually I didn't change the code, but u are right, it should be 2048, but the source data size is [32, 1000], see the image down below, I add one line `print("ResNet.py, line188: ", source.size())` before the `source = self.cls_fc(source)`. The dataset is downloaded from the URL given in the README.md.
![image](https://user-images.githubusercontent.com/18626699/79094665-d1f9f680-7d8a-11ea-9385-777d776bdf3e.png)

**Comment by easezyc** [comment]
<img width="466" alt="微信图片_20200413133738" src="https://user-images.githubusercontent.com/20279673/79095094-18038a00-7d8c-11ea-8781-81418f8e80e6.png">
Note line 168 in Resnet.py

**Comment by VoiceBeer** [comment]
I'm so sorry xD. Due to the network issue on my server, I have to download the pth file and upload it to the server. After checking all the files and codes, I just found that I downloaded the ResNet18 model, what a embarrassed mistake :<, so sorry to bother u, now everything works fine and no need to change any code or file.

BTW, I think the solution in issue #4 should be added into the README.md to help others. Really useful.

Anyway, thx a lot! And good day :>.
